pkg/machines/scripts/create_machine.sh

```diff
@@ -18,16 +18,20 @@ handleFailure(){
 }
 
 spiceSupported(){
-    # map system architecture to qemu-system-* command
+    # map system architecture to qemu-system-* or qemu-kvm command
     ARCH=$(uname -m)
     case "$ARCH" in
         i?86) QEMU=qemu-system-i386 ;;
         ppc64*) QEMU=qemu-system-ppc64 ;;
         *) QEMU=qemu-system-$ARCH;
     esac
 
-    printf '{"execute":"qmp_capabilities"}\n{"execute":"query-spice"}\n{"execute":"quit"}' | \
-        $QEMU --qmp stdio --nographic -nodefaults  | grep -q '"enabled":'
+    type -P $QEMU > /dev/null || QEMU=qemu-kvm
```

Comment:
This will not work because qemu-kvm is in /usr/libexec which is not by default in $PATH.

Reply:
Also, I think by not putting it into `$PATH` the packagers of qemu-kvm don't want it to be used directly. On Fedora, qemu-kvm is a metapackage that pulls in qemu-system-x86_64. If it's different somewhere else (like on RHEL 7 apparently), then let's just disable spice there as well. This is safer IMHO.

Reply:
ok, so let's postpone SPICE configuration on such a system to extra step after VM creation (UI for that is not available, but planed).
I'll remove the qemu-kvm from here.